lib/caerbannog/queue.rb

```diff
@@ -8,8 +8,9 @@ def self.push(name, payload)
       @message_class.create!(:name => name, :payload => JSON.generate(payload))
     end
 
-    def self.rabbitmq
-      Bunny.run ENV['RABBIT_URL'] do |conn|
+    def self.rabbitmq(rabbit_url = nil)
+      rabbit_url ||= ENV['RABBIT_URL']
```

Comment:
I would vote for adding `self.rabbitmq_read_url=` and `self.rabbitmq_write_url` to this class instead of relying on ENV variables.